workshops/orchestrator_course/prefect_bckfil_flow.py

```diff
@@ -41,28 +44,27 @@ def run_resource(resource_name: str, md_db: str, start_date: str = None, end_dat
     """
     # Ensure environment vars for dlt are set for this task execution
     _set_env(md_db)
-    import example_pipeline,copy
     # Copy pipeline config so modifications don’t leak between tasks
-    cfg = copy.deepcopy(example_pipeline.config)
+    config = copy.deepcopy(example_pipeline.config)
     # If the resource is "forks" and a backfill window is provided, inject it dynamically
     if resource_name == "forks" and start_date and end_date:
-        for res in cfg["resources"]:
+        for res in config["resources"]:
             if res["name"] == "forks":
                 res["endpoint"]["incremental"]["initial_value"] = start_date
                 res["endpoint"]["incremental"]["end_value"] = end_date
     # pick just one resource from your dlt source
-    src = rest_api_source(cfg).with_resources(resource_name)
+    source = rest_api_source(config).with_resources(resource_name)
```

Comment:
this is a bit all over the place:
`pipeline_example.py` you define the github_source
`dlt_pipeline.py` also defines it
you then use neither of the above and instead take config and instantiate the `rest_api_source` with it

I think your files need to be named after what they define, you can include an example way to run the source either commented out, or what's cleaner in a __main__ clause so that it wont be executed if users import the file

here you should then .with_resources, which returns a clone of the original source with just your resource selected:
`source = github_source(config).with_resources(resource_name)`